packages/visual-editor/src/components/Dropzone/useComponentProps.ts

```diff
@@ -240,10 +240,12 @@ const addExtraDropzonePadding = (padding: string) =>
   padding
     .split(' ')
     .map((value) => {
-      if (value.endsWith('px')) {
-        const parsedValue = parseInt(value.replace(/px$/, ''), 10);
-        return (parsedValue < DRAG_PADDING ? DRAG_PADDING : parsedValue) + 'px';
+      const numericValue = parseFloat(value);
+      const unit = value.replace(numericValue.toString(), '');
+
+      if (numericValue === 0 || (unit === 'px' && numericValue < 4)) {
```

Comment:
I don't think this is a long term solution when margin and padding gets updated soon to have the new input units. What happens when a user selects 1em?

Reply:
Not exactly.  This will only change padding to 4px if the parsed value equals zero, or the units are pixels and the parsed value is less than 4

Reply:
You will have the reverse issue*, that it won't recognize other units when they are <4px when converted. I'd say we move forward with this but we create a ticket to track this issue

Reply:
I committed a update and changed this to use `calc()` so that the padding will be more consistently increased by 4px for other units like %, em, rem, etc.  The additional padding is subtle when dragging so I think we can try this out and see if there's any feedback that would cause us to reconsider this approach